wiki/Beatmap/Pattern/osu!mania/Trill/en.md

```diff
@@ -0,0 +1,27 @@
+---
+stub: true
+---
+
+# Trill
+
+A **Trill** is three or more consecutive notes that alternate between two different columns. 
+
+![](img/trill.png)
+
+**Trills** are often characterized based on whether they require two hand movements or one. These are known as **one-handed trills** and **two-handed trills**.
+
+## Chordtrill, jumptrill and split-trill
```

Comment:
Chordtrill doesn't really get used much unless it's 7k and that's still not the most common way to refer to them (some people refer to them as splittrills or jumptrills unrelated to the chordsize but because of the motion they require). I included chordtrill anyways since it's still technically a term and mania RC also uses it so I wanted to also standardize with that. I only grouped the other two together because of how similar jumptrills and splitrills are in theory since they both use jumps, which I thought would be easier navigation wise.